File: Beecrowd/8-Geometria_Computacional/2830-Balao++-2/2830.js
const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout
});

readline.question("", input => {
    const [radix, gas] = input.split(" ").map(Number);
    const area = (4.0 / 3.0) * (3.1415 * (radix ** 3));
    console.log(Math.floor(gas / area));

});

File: Beecrowd/8-Geometria_Computacional/2830-Balao++-2/2830.cs
using System;


class Program {
    public static void Main(string[] args) {
        string[] data = Console.ReadLine().Split(' ');
        double radix= double.Parse(data[0]);
        double gas = double.Parse(data[1]);

        double area =  (4.0/3.0) * (3.1415*(radix*radix*radix));
        Console.WriteLine((int)(gas/area));
    }
}
